theano/misc/check_multi_gpu.py

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env python
```

Comment:
I'm not able to use this file. If I use dev as gpu0, gpu2, or cuda0 cuda1 cuda2, it do not work:

```
$PYTHONPATH=.:$PYTHONPATH python theano/misc/check_multi_gpu.py cuda2 cuda0
Traceback (most recent call last):
  File "theano/misc/check_multi_gpu.py", line 50, in <module>
    main(sys.argv[1], sys.argv[2])
  File "theano/misc/check_multi_gpu.py", line 14, in main
    ctx2 = pygpu.init(dev2)
  File "gpuarray.pyx", line 580, in pygpu.gpuarray.init (pygpu/gpuarray.c:7015)
  File "gpuarray.pyx", line 551, in pygpu.gpuarray.pygpu_init (pygpu/gpuarray.c:6953)
  File "gpuarray.pyx", line 955, in pygpu.gpuarray.GpuContext.__cinit__ (pygpu/gpuarray.c:10318)
pygpu.gpuarray.GpuArrayException: Invalid device ordinal
```